app/api/auth/authMiddleware.ts

```diff
@@ -0,0 +1,25 @@
+import { Request, Response, NextFunction } from 'express';
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import { User } from '../users/usersModel';
```

Comment:
Please, refer to this comment about the importing User type:
https://github.com/huridocs/uwazi/pull/2913/files#r425370222

Reply:
I prefer to have the code the way we develop everything else and disable a rule that is clearly bugged.